schemas.py

```diff
@@ -0,0 +1,56 @@
+from pydantic import BaseModel
+from datetime import date, datetime
+
+
+class BookBase(BaseModel):
+    title: str
+    summary: str | None = None
+    publication_date: date
+
+    class Config:
+        orm_mode = True
+
+
+class AuthorBase(BaseModel):
+    name: str
+    bio: str | None = None
+
+    class Config:
+        orm_mode = True
+
+
+class BookCreate(BookBase):
+    author_id: int
```

Comment:
what happens if i not provide `summary` during book creation ?

Reply:
i thought it should be allright since it wasnt a mandatory field according to the task. But right now looking at my code im not sure. Will it be ok if make it nullable?